ikarus/python/finiteelements/fe.hh

```diff
@@ -100,6 +100,8 @@ void registerFE(pybind11::handle scope, pybind11::class_<FE, options...> cls) {
           pybind11::keep_alive<1, 2>());
 
   cls.def("bind", [](FE& self, const GridElement& e) { self.bind(e); });
+  cls.def("updateState",
+          [](FE& self, const Requirement& req, const Eigen::Ref<Eigen::VectorXd> dx) { updateState(self, req, dx); });
```

Comment:
This const at the refe should not be there. It should `Eigen::Ref<const Eigen::VectorXd> `  shouldn't it. Maybe there is some pybind11 issue I dont see.